packages/manager/modules/pci/src/projects/project/storages/databases/add/add.constants.js

```diff
@@ -11,10 +11,22 @@ export const ORDER_KEYS = [
   'networkId',
   'subnetId',
 ];
+export const PRIVATE_NETWORK_GUIDE = {
+  DEFAULT: 'https://docs.ovh.com/gb/en/public-cloud/public-cloud-vrack/',
+  ASIA: 'https://docs.ovh.com/asia/en/public-cloud/public-cloud-vrack/',
+  AU: 'https://docs.ovh.com/au/en/public-cloud/public-cloud-vrack/',
+  CA: 'https://docs.ovh.com/ca/en/public-cloud/public-cloud-vrack/',
+  FR: 'https://docs.ovh.com/fr/public-cloud/public-cloud-vrack/',
+  GB: 'https://docs.ovh.com/gb/en/public-cloud/public-cloud-vrack/',
+  IE: 'https://docs.ovh.com/ie/en/public-cloud/public-cloud-vrack/',
+  QC: 'https://docs.ovh.com/ca/fr/public-cloud/public-cloud-vrack/',
+  SG: 'https://docs.ovh.com/sg/en/public-cloud/public-cloud-vrack/',
```

Comment:
No guides for IT/DE/ES/PT etc..?

Reply:
Good point.
To build this list of private network guide we actually took what was available done for the API guides (see https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/project.constants.js).
In order to not delay the merge of this feature, I will create a new feature in our backlog to update the link to both the API guides and the private network guides. In the meantime if the select subsidiary is either IT/DE/ES/PT the link will point to the DEFAULT key which is english, which is fine.
Is that OK for you ?

Reply:
ok for me